command/sub8_missions/missions/align_channel.py

```diff
@@ -10,49 +12,53 @@
 
 import numpy as np
 
+SEARCH_DEPTH = .65
+
 
 @util.cancellableInlineCallbacks
 def run(sub_singleton):
     global marker_found
 
-    print "Searching"
     nh = sub_singleton._node_handle
-    next_pose = yield nh.get_service_client('/next_search_pose', SearchPose)
+    #next_pose = yield nh.get_service_client('/next_search_pose', SearchPose)
     cam = PinholeCameraModel()
 
     # Go to max height to see as much as we can.
-    yield sub_singleton.move.depth(0.6).zero_roll_and_pitch().go()
+    #yield sub_singleton.move.depth(SEARCH_DEPTH).zero_roll_and_pitch().go()
     yield nh.sleep(1.0)
 
+    # ---------------------------------------
+    # MOST OF THIS IS FOR THE SEARCH PATTERN.
+    # ---------------------------------------
     # This is just to get camera info
     resp = yield sub_singleton.channel_marker.get_2d()
     cam.fromCameraInfo(resp.camera_info)
 
-    intial_pose = sub_singleton.last_pose()
-    intial_height = yield sub_singleton.get_dvl_range()
-    raidus = calculate_visual_radius(cam, intial_height)
```

Comment:
Could have an option to return the pose from the ground where it changed the z of the current pose.